OUXT-Polaris/robotx_packages
# Develop

## Status
**DEVELOPMENT**

## Description
add move_base to master blanch

## Todos

## Steps to Test or Reproduce

```
roslaunch robotx_gazebo robotx_gazebo.launch
```
See also
[![No video](https://img.youtube.com/vi/rgQ5Se_MXUo/0.jpg)](https://www.youtube.com/watch?v=rgQ5Se_MXUo)
